geo/fuzz/Cargo.toml

```diff
@@ -12,9 +12,11 @@ cargo-fuzz = true
 libfuzzer-sys = "0.4"
 
 [dependencies.geo]
+path = ".."
```

Comment:
specifying a dependency without a version or path was causing a warning (which became an error in rust-1.55).